mono/eglib/goutput.c

```diff
@@ -122,8 +139,11 @@ g_logv_nofree (const gchar *log_domain, GLogLevelFlags log_level, const gchar *f
 {
 	char *msg;
 
-	if (g_vasprintf (&msg, format, args) < 0)
+	if (internal_abort_func) {
+		msg = NULL;
```

Comment:
True, that would make this easier to handle. It seemed a bit much to introduce a dependency from eglib onto mono-threads. 

I can move it all into eglib when I make it an inline function.

Reply:
Since we are increasing its use over the runtime, it seems good to move to eglib. You can take this chance to make it an `static inline` function for https://github.com/mono/mono/pull/12125#discussion_r247235469